spaceship.zsh

```diff
@@ -446,11 +448,19 @@ spaceship_host() {
 spaceship_dir() {
   [[ $SPACESHIP_DIR_SHOW == false ]] && return
 
-  _prompt_section \
-    "$SPACESHIP_DIR_COLOR" \
-    "$SPACESHIP_DIR_PREFIX" \
-    "%${SPACESHIP_DIR_TRUNC}~" \
-    "$SPACESHIP_DIR_SUFFIX"
+  if _is_git && [[ $SPACESHIP_DIR_TRUNC_IN_GIT == true ]]; then
+    _prompt_section \
+      "$SPACESHIP_DIR_COLOR" \
+      "$SPACESHIP_DIR_PREFIX" \
+      "$(basename $(git rev-parse --show-toplevel))" \
```

Comment:
Yeah, thats also what i noted in the PR starter. Don't really like that. It should be from git root directory? Don't really know how to do it yet.